ksonnet-gen/astext/astext_test.go

```diff
@@ -47,6 +49,44 @@ func TestCreateField(t *testing.T) {
 						Kind:  ast.StringDouble,
 					}}},
 		},
+		{
+			name: "__leading",
+			expected: &ObjectField{
+				ObjectField: ast.ObjectField{
+					Kind: ast.ObjectFieldID,
+					Id:   &leadingID,
+				}},
+		},
+		{
+			name: "dot.name",
+			expected: &ObjectField{
+				ObjectField: ast.ObjectField{
+					Kind: ast.ObjectFieldStr,
+					Expr1: &ast.LiteralString{
+						Value: "dot.name",
+						Kind:  ast.StringDouble,
+					}}},
+		},
+		{
+			name: ".",
+			expected: &ObjectField{
+				ObjectField: ast.ObjectField{
+					Kind: ast.ObjectFieldStr,
+					Expr1: &ast.LiteralString{
+						Value: ".",
+						Kind:  ast.StringDouble,
+					}}},
+		},
+		{
+			name: "9p",
+			expected: &ObjectField{
+				ObjectField: ast.ObjectField{
+					Kind: ast.ObjectFieldStr,
+					Expr1: &ast.LiteralString{
+						Value: "9p",
```

Comment:
please tell me this is a subtle plan9 filesystem protocol reference

Reply:
😊